packages/rps-poc/README.md

```diff
@@ -71,7 +71,7 @@ This will fire up the Storybook panel inside a browser.
 
 #### To run smart contract tests:
 
-`yarn run test:truffle`
+`yarn run test:integration`
```

Comment:
@lalexgap Do you think we should change the naming scheme? If I understand correctly, these aren't really integration tests for the app. They are integration tests for the smart contracts. So maybe `yarn run test:contracts`?

Reply:
@kerzhner that makes sense to me.